<launch>
    <node pkg="imu_nav" type="dead_reckoning_demo" name="dead_reckoning" output="screen">
    <param name="imu_topic" type="str" value="/imu0" />
    <param name="imu_coordinate_system" type="str" value="NED" />
    <param name="apply_calibration" type="bool" value="true" />
    <param name="use_dominant_velocity_model" type="bool" value="false" />
    <param name="imu_calib_file" type="str" value="/home/dhl/git/catkin_ws/src/imu_nav/launch/imu_calib-2018-10-27-ellipse.yaml" />
    <!-- imu_calib-2018-10-23-ellipse.yaml -->
    <!-- imu_calib-2018-10-27-ellipse.yaml -->
    <!-- imu_calib-2018-10-22-17-47-02.yaml -->
    </node>
</launch>